base_iso3166/static/description/index.html

```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Why removing this?

Reply:
`DeprecationWarning: XML declarations in HTML module descriptions are deprecated since Odoo 17, account_financial_report can just have a UTF8 description with not need for a declaration.`

Reply:
But in some pre-commit modules it is adding it

Reply:
Yes, but it's also removed on the XML file, not only on the HTML.

Reply:
I added it again, check the tests, you will see the warining in "Run tests" section...

Reply:
Yeah, I mean in the XML file, not in the HTML file. Anyway, if it's added by the readme generator, we need to adjust such generator to not add the XML header in this file. cc @sbidoul @yajo @legalsylvain

Reply:
Yes, I misunderstood, it should not be deleted in XML, only in HTML, I have already added it again

Reply:
I think the readme generator will re-add it on merge. This is an xhtml file, so that should be ok.

Reply:
@sbidoul But odoo 17 says that it is deprecated, we would have to modify the readme generator so that it does not add it, otherwise the odoo warning log will be filled.

In this specific module the index.html file is not generated by the generator. In others yes and when executing pre-commit it adds it again.

Reply:
> odoo 17 says that it is deprecated

@ramiadavid Can you point me the warning (or better the code in Odoo that emits that deprecation)?

> In this specific module the index.html file is not generated by the generator

How so?

Reply:
https://github.com/OCA/maintainer-tools/blob/f71041f22b8cd68cf7c77b73a14ca8d8cd190a60/tools/gen_addon_readme.py#L403

Reply:
@ramiadavid that's not the Odoo core code emitting the error.

Reply:
I'm going to review it, but I doubt it, you just have to read the error DeprecationWarning: XML declarations in HTML module descriptions are deprecated since **Odoo 17**, base_iso3166 can just have a UTF8 description with not need for a declaration.

Reply:
@pedrobaeza @sbidoul 
https://github.com/odoo/odoo/blob/bbb4a2e08adc1c93a4d65344fec01b69c8e2a744/odoo/addons/base/models/ir_module.py#L170

Reply:
Thanks, the README generator should be changed then. Meanwhile, I think we can continue with the warning.